contrib/kafka/config/broker/kafka-broker.yaml

```diff
@@ -1,19 +1,32 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 ########################################## KAFKA BROKER ######################################
 # The following does not need to live in the same namespace as the bus.
----
+
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: kafka-broker
 spec:
   replicas: 1
   selector:
-    matchLabels:
+    matchLabels: &labels
```

Comment:
This ref is only valid for this document (between `---` fences), I presume? (Your testing would presumably have caught otherwise.)

Reply:
Yes, much to my chagrin when I want to match a `Service` and a `Deployment`.